src/middlewares/auth.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const authentication = (req, res, next) => {
+  const bearerHeader = req.headers.authorization;
```

Comment:
I feel we can check for other places where the token can be placed in the request object like `req.body.token`, `req.query.token`

Reply:
Also, I prefer we use `x-access-token` custom header for setting the token